web/thesauruses/kotlin/1.5/classes.json

```diff
@@ -0,0 +1,112 @@
+{
+    "meta": {
+      "language": "kotlin",
+      "language_version": "1.5",
+      "language_name": "Kotlin",
+      "structure": "classes"
+    },
+    "concepts": {
+      "normal_class": {
+        "code": "class ClassName {\n // Class body containing properties and methods \n}",
+        "name": "Normal class"
+      },
+      "abstract_class": {
+        "code": "abstract class ClassName {\n // Class body containing abstract methods \n}",
+        "name": "Abstract class"
+      },
+      "interface": {
+        "code": "interface InterfaceName {\n // Interface body containing method declarations \n}",
+        "name": "Interface"
+      },
+      "read_only_class": {
+        "code": "class ClassName(val property: Type) {\n // Only getter methods are provided \n}",
+        "name": "Read-only class"
+      },
+      "static_class": {
+        "code": "class ClassName {\n companion object {\n // Static properties and methods \n}\n}",
```

Comment:
On some concepts you have `\n}\n}`, which shows up all left-aligned:
![image](https://github.com/user-attachments/assets/e0a58146-caae-41f2-a08f-32006ea7619a)

Do you want to add in formatting so the inner `}` is indented? If so, I'll let you edit it. If not, I'll merge it.

(This applies to a few other concepts here too)

Reply:
I have removed the unnecessary comment from the `implement_deconstructor` concept and fixed the formatting for the code snippets to ensure proper indentation.